graphql/queries.graphql

```diff
@@ -1,24 +1,23 @@
 # NOTICES
 
-query notice($noticeIndex: Int!, $inputIndex: Int!) {
-  notice(noticeIndex: $noticeIndex, inputIndex: $inputIndex) {
+query notice($outputIndex: Int!) {
+  notice(outputIndex: $outputIndex) {
     index
     input {
+      id
       index
+      payload
     }
     payload
     proof {
-      validity {
-        inputIndexWithinEpoch
-        outputIndexWithinInput
-        outputHashesRootHash
-        vouchersEpochRootHash
-        noticesEpochRootHash
-        machineStateHash
-        outputHashInOutputHashesSiblings
-        outputHashesInEpochSiblings
-      }
-      context
+      outputIndex
```

Comment:
We will need to update this in the future

Reply:
Indeed
Let's make always synchronized with nonodo version